users/config.rst

```diff
@@ -591,6 +591,10 @@ Listen Addresses
 
 The following address types are accepted in sync protocol listen addresses:
 
+Default listen addresses (``default``)
+    This is equivalent to ``tcp://0.0.0.0:22000`` and
```

Comment:
The meaning is the same, your variant is in "code/input field" notation while mine is written out in a sentence. The notation should probably be mentioned in the first paragraph, e.g. by writing

    The following address types are accepted in sync protocol listen addresses and multiple addresses can be entered separated by ", ":